modules/packages/URL.chpl

```diff
@@ -147,8 +149,10 @@ module URL {
     use Curl;
     use CurlQioIntegration;
     var f = openCurlFile(url, iomode.cw, style);
-    return f.writer(kind=kind, locking=locking,
-                    region=start..end);
+    // TODO: change this back to f.writer when the fromOpenUrlWriter arg is
+    // removed
+    return f.writerHelper(kind=kind, locking=locking,
+                          region=start..end, fromOpenUrlWriter=true);
```

Comment:
should this region argument also be `start..#end`?

Reply:
Maybe `openURLWriter` needs a test as well? I'm not seeing one in the test directory

Reply:
Oops, yes!

Reply:
There are tests of openUrlWriter in `test/library/packages/Curl`, they just require Curl is installed (which isn't true on all of our testing boxes).  So I suspect you just saved me from a nightly failure, but that we would have caught it then

Reply:
Oh that makes sense. Glad I saw that!